src/model/CoolingTowerVariableSpeed.cpp

```diff
@@ -748,6 +749,22 @@ namespace model {
       }
     }
 
+    ComponentType CoolingTowerVariableSpeed_Impl::componentType() const {
+      return ComponentType::Cooling;
+    }
+
+    std::vector<FuelType> CoolingTowerVariableSpeed_Impl::coolingFuelTypes() const {
+      return {FuelType::Electricity};
+    }
+
+    std::vector<FuelType> CoolingTowerVariableSpeed_Impl::heatingFuelTypes() const {
+      return {};
+    }
+
+    std::vector<AppGFuelType> CoolingTowerVariableSpeed_Impl::appGHeatingFuelTypes() const {
+      return {AppGFuelType::Electric};
```

Comment:
The other cooling tower objects don't have AppGFuelType::Electric specified in the return.  Either way is fine; just think they should be consistent.

Reply:
Yeah, cooling towers should be Electric

Reply:
Done in 0a4c4324aaa10372d723b6cdf685a8883481092c